docs/usage/app-hooks.rst

```diff
@@ -1,3 +1,82 @@
 =================
```

Comment:
So, I recommend we don't use the double under/over thing to designate different levels of sections, but instead the [standard]https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections) recommended by Sphinx. This is also how the automated tooling I've been using (pandoc/m2r) handle this. 

I'll see if we can add a linter of some sort to enforce this.

Reply:
Is there an example `.rst` file I can refer to for a better understanding of what you've in mind?

Reply:
So, is this what you've have in mind?

```restructuredtext
Chapter Headline
****************

Some content....

Chapter Subsection
------------------

Some content about the subsection....
```

Reply:
Yup. Since the characters used can be any punctuation, it's really just a convention, and we could make up our own, but I'd say it's best we simply follow an established one.